### ISSUE TITLE
Support `css` option in config to solve Font Awesome Icon loading incorrectly (very large)

### ISSUE THREAD

**Issue by samanthaming** [vuejs/vuepress#2182]
So I found out on my site, the font awesome icons loads super big initially and snaps back into proper size after (thank you @Maybach91 for letting me know 👍) 

![image](https://user-images.githubusercontent.com/16507205/74560829-44ca2b00-4f1c-11ea-94c0-4459bdcc9bd0.png)

### Why is this happening?

Check out the response from @robmadole, he explains it in this issue:

https://github.com/FortAwesome/vue-fontawesome/issues/14#issuecomment-342242540

### Feature Request

I think I have a fix, but Nuxt seems to have a better idea of solving this:

https://github.com/FortAwesome/vue-fontawesome/issues/14#issuecomment-438245880

![image](https://user-images.githubusercontent.com/16507205/74561508-87403780-4f1d-11ea-8112-876a961e18d7.png)

Maybe VuePress can also accept a `css` option in their `config` file?

### My Current Solution

This is my current solution and it seems to work for me 🤞(If it doesn't, please let me know!)

_enhanceApp.js_

```javascript
import { config } from '@fortawesome/fontawesome-svg-core';

config.autoAddCss = false;
```

And then add the fa style in either inside your `styles` folder (I'm using stylus, but it should also work for `scss` files)

```css
@import '~@fortawesome/fontawesome-svg-core/styles.css';
```

OR in your vue component `.vue`

```html
<style>
  @import '~@fortawesome/fontawesome-svg-core/styles.css';
</style>
```

**Comment by meteorlxy** [comment]
See https://github.com/FortAwesome/vue-fontawesome/issues/86

**Comment by robmadole** [comment]
@samanthaming yeah, this is a good solution. would you be interested in putting up a PR that adds VuePress? https://github.com/FortAwesome/vue-fontawesome#integrating-with-other-tools-and-frameworks
